packages/extension-ui/src/Popup/Derive/Derive.test.tsx

```diff
@@ -150,13 +160,31 @@ describe('Derive', () => {
       expect(wrapper.find('.error')).toHaveLength(0);
     });
 
-    it('"Create derived account" is disabled when suri is incorrect', async () => {
+    it('An error is visible and "Create derived account" is disabled when suri is incorrect', async () => {
+      await type(wrapper.find('input[type="password"]'), 'pass');
+      await type(wrapper.find('[data-input-suri] input'), '//');
+      wrapper.find('[data-button-action="create derived account"] button').simulate('click');
+      await act(flushAllPromises);
+      wrapper.update();
+
+      const button = wrapper.find('[data-button-action="create derived account"] button');
+
+      expect(button.prop('disabled')).toBe(true);
+      expect(wrapper.find('.error')).toHaveLength(1);
+    });
+
+    it('The error disappears and "Create derived account" is enabled when typing a new suri', async () => {
       await type(wrapper.find('input[type="password"]'), 'pass');
       await type(wrapper.find('[data-input-suri] input'), '//');
+      wrapper.find('[data-button-action="create derived account"] button').simulate('click');
+      await act(flushAllPromises);
+      wrapper.update();
+      await type(wrapper.find('[data-input-suri] input'), 'new');
 
       const button = wrapper.find('[data-button-action="create derived account"] button');
 
-      expect(button.prop('disabled')).not.toBe(true);
```

Comment:
This test was no testing the current flow because there was no click on the "create derivation" button, hence no validation for the suri.